components/Blog/Card/Info.tsx

```diff
@@ -0,0 +1,16 @@
+interface Props {
+   children: React.ReactNode & {
+      [index: number]: string;
+   };
+}
+
+const Info = ({ children }: Props) => {
+   return (
+      <div className="text-sm font-semibold text-indigo-700">
+         <span>{children[0]}</span>
```

Comment:
it does not seem like a healty way to do that, is not it kinda over engineering?